### DISCUSSION TITLE
Sequence to chat with a contact through the API

### DISCUSSION THREAD

**Discussion by jeovajireh** [chatwoot/chatwoot#1572] [Q&A]
Hello everybody.

I can create new contacts through the API, however, whenever I send a message via the API it is always assigned to the contact Jane, What would be the correct order to create a new contact, start a conversation with an agent through this new contact using the API ?

1 - Create contact

curl --request POST 'http://127.0.0.1:3003/api/v1/accounts/1/contacts' --header 'Content-Type: application / json' --header 'api_access_token: NQfhAnM58ThYybXV119BYrLR' --data- raw '{"name": "RBS", "email": "aa@aa.com", "phone_number": "912345678"}'

2 - Start a conversation with the contact created above.

3 - Send message to an agent using the contact created above

Thanks for any help

**Comment by pranavrajs** [comment]
@viapratica Please check https://www.chatwoot.com/docs/channels/api/send-messages and see if that solves your query.

**Reply by jeovajireh** [reply]
Congratulations on your beautiful work on this project! Thank you for the very quick response.
